FinancialTransactionsApi/V1/Controllers/FinancialTransactionsApiController.cs

```diff
@@ -203,8 +208,7 @@ public async Task<IActionResult> Update([FromHeader(Name = "x-correlation-id")]
             {
                 return BadRequest(new BaseErrorResponse((int) HttpStatusCode.BadRequest, GetErrorMessage(ModelState)));
             }
-
-            var existTransaction = await _getByIdUseCase.ExecuteAsync(id).ConfigureAwait(false);
+            var existTransaction = await _getByIdUseCase.ExecuteAsync(id, transaction.TargetId).ConfigureAwait(false);
```

Comment:
Should the use case used here be _getAllUseCase or _getTransactionListUseCase instead?

Reply:
No, we are only fetching single record

Reply:
Hi Edward, do you mind replying to my query please as this thread should not be resolved.